mobile/evidence/ios/Leaf/Sources/Leaf/Components/LeafAvatar.swift

```diff
@@ -1,15 +1,7 @@
-//
-//  AvatarView.swift
-//  Evidence
-//
-//  Created by Cris Messias on 28/09/23.
-//
-
 import SwiftUI
-import CachedAsyncImage //Package Dependencies
-
+import CachedAsyncImage
 
-struct AvatarView: View {
+struct LeafAvatar: View {
```

Comment:
This changes are already on `main` branch. You need to sync your branch with `main` using merge/rebase.

Reply:
I have 0 idea why these chances are in these pull requests, but I hope this is out after I since the mains remote/local